packages/gatsby-theme-docs/README.md

```diff
@@ -152,6 +154,67 @@ The available JSX components are:
 
 > When using JSX components, it's recommended to leave a **blank line** between the element tags and the actual content. This allows the content to be parsed as markdown, so you can use markdown syntax within the custom component tags.
 
+## Using Theme with Add-Ons
+
+A theme add-on is a Gatsby Theme that exposes React components to be injected into the MDX provider of the core theme.
+
+Usually it's possible for a child theme to use components shadowing (see [Theme overrides](#theme-overrides)). However, with multiple themes, the shadowed components are loaded from the last theme child. To solve this problem, a commercetools-docs Gatsby Theme can be used as an add-on.
+
+When using add-on themes, a proxy export file will be created in the websites `src/@commercetools-docs/gatsby-theme-docs/overrides` folder to leverage Gatsby's component shadowing (see [Theme overrides](#theme-overrides)), including all the exported components from the add-on packages.
+
+**We recommend to use the `configureThemeWithAddOns` function to configure the websites's `gatsby-config.js`**
+
+```js
+const {
+  configureThemeWithAddOns,
+} = require('@commercetools-docs/gatsby-theme-docs/configure-theme');
+
+module.exports = {
+  plugins: [
+    ...configureThemeWithAddOns({
+      // Pass the normal theme options
+      websiteKey: 'my-website-key',
+      // Define the add-on plugins
+      addOnPlugins: [
```

Comment:
it's really confusing that depending on the form of configuration the parameter has a different name.  The `configureThemeWithAddOns`  function would be possible also if both variations were just called `addOns` - the function could still handle it properly.

Reply:
Correct. Now that we don’t document the “alternative” approach anymore, we can use the same value, otherwise it would have been confusing.